### ISSUE TITLE
Animation issues

### ISSUE THREAD

**Issue by Quinten123** [julienduroure/gltf2-blender-importer#24]
Hi, here is an example file that doesnt correctly import animations. The rotations are all weird and the bones aren't connected to eachother. Another issue is that when there is more than 1 mesh, vertex groups are only applied to 1 mesh.

Example file: [https://drive.google.com/drive/folders/16NcfutqTC3TRRWDKLewyxxfaHYJV2G-R?usp=sharing](url)

**Comment by julienduroure** [comment]
Hello,
* vertex group issue --> now fixed in master
* bone issue --> This is a known issue, centralized in #20  (Working on it !)

**Comment by julienduroure** [comment]
Also note that there are in this gltf some extension not implemented here (like LOD).
This file also refers to some image file that are not present (link to LOD ?)

**Comment by Quinten123** [comment]
Yes, i think it refers to lower resolution textures. I also have some gltf files like these where it refers to different model LOD's as well.
